hack/remove-calico-policy/iptables-remover-ds.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: calico-remove-iptables
+  namespace: kube-system
+  labels:
+    k8s-app: calico-remove-iptables
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-remove-iptables
+    spec:
+      hostNetwork: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: projectcalico.org/ds-ready
+                  operator: NotIn
+                  values:
+                  - "true"
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      containers:
+        - name: calico-remove-iptables
+          image: calico/iptables-remover
```

Comment:
It's not a step in the process - it's not part of the "Instructions" section.

If there are Calico components that don't have instructions on how to build them, then those docs are wrong and need to be updated rather than removing the instructions from here.